### DISCUSSION TITLE
OpenAI Codex Model Upgrade

### DISCUSSION THREAD

**Discussion by BeamoINT** [community/community#77057] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

There should be a model update to OpenAI Codex so that instead of being based on GPT-3, it should be based on GPT-4 where it is hyper optimized for coding when it comes to chat as well as it should have a completions ability where it is a much more powerful version of Codex.

**Comment by outgaze** [comment]
> ### Select Topic Area
> Product Feedback
> 
> ### Body
> There should be a model update to OpenAI Codex so that instead of being based on GPT-3, it should be based on GPT-4 where it is hyper optimized for coding when it comes to chat as well as it should have a completions ability where it is a much more powerful version of Codex.

i hope you mean gpt-4-turbo cause the basic gpt-4 is still with a cutoff knowledge date of 2021 while turbo is 2023 april

**Reply by BeamoINT** [reply]
I do, that means it will also have a 128k context window if they make it.
